tools/check-schemas

```diff
@@ -71,6 +71,9 @@ def get_event_checker(event: dict[str, Any]) -> Callable[[str, dict[str, Any]],
     # And add the suffix.
     name = name + "Event"
 
+    if name == "DeleteMessageEvent":
+        name = "DeleteStreamMessagesEvent"
```

Comment:
This seems like an odd construction. It should get a comment at least, but ideally wouldn't need to exist.

Reply:
The previous version of `DeleteMessageEvent` got split out into four new classes.  We only have one example in our JS fixtures that needs to be verified, and it's the `DeleteStreamMessagesEvent` flavor`.  I chose to use the two-line hack here vs. creating a union type on the Python side.  The full commit puts this more into context.  I'm open to other ideas here.

If you agree that the two-line hack here is the most expedient solution, I will basically just put some version of this PR comment into the code.  I actually think there's probably a better solution here.  Maybe not for this specific problem (it's hard to argue with a simple 2-line hack) but holistically.

Reply:
Ahh, OK -- yeah just add a small comment, we can clean it up later once we've improved the JS fixtures.

Reply:
Added actually kind of a long comment here, actually, but feel free to edit it down.